src/Jackalope/ObjectManager.php

```diff
@@ -818,12 +818,12 @@ public function save()
             $this->executeOperations($this->operationsLog);
 
             // loop through cached nodes and commit all dirty and set them to clean.
-            if (isset($this->objectsByPath['Node'])) {
-                foreach ($this->objectsByPath['Node'] as $node) {
+            foreach ($this->objectsByPath as $nodes) {
```

Comment:
Would jackrabbit allow this? I thought the version nodes should not be updated manually...